deel-2-flexbox-layout/opdracht-1/styles/styles.css

```diff
@@ -1 +1,35 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+
+body {
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+}
+
+header {
+    flex: 0 0 15vh;
+}
+
+main {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    flex: 1 0 auto;
+}
+
+article {
+    order: 1;
+    flex: 1 1 80vh;
+}
+aside {
+    order: 2;
+    flex: 1 1 10vh;
+}
+nav {
+    order: 3;
+    flex: 1 1 10vh;
+}
+
+footer {
```

Comment:
Footer staat in een column container, daarom kan het beter een vaste waarde hebben. Dit wordt aangeraden vanuit de readme: 
TIP: je kunt de waarde van flex-basis in een column-container beter uitdrukken in absolute waardes ( zoals pixels) en in een row-container beter in relatieve waardes (zoals vw of procenten). Voor elementen die geen vaste hoogte/breedte hebben (zoals vaak bij het main element het geval is), mag je de waarde auto gebruiken.